src/gleam/http/cookie.gleam

```diff
@@ -0,0 +1,116 @@
+import gleam/int
+import gleam/list
+import gleam/option.{Option, Some}
+import gleam/string
+
+const epoch = "aa"
+
+pub type SameSitePolicy {
+  Lax
+  Strict
+  None
+}
+
+pub type Attributes {
+  Attributes(
```

Comment:
Defined as a Single record rather than a list of options.

I think this makes sense because the Spec explicitly states

>   6.  Process the attribute-name and attribute-value according to the
       requirements in the following subsections.  (Notice that
       attributes with unrecognized attribute-names are ignored.)

https://tools.ietf.org/html/rfc6265#section-5.2

I took inspiration from a kotlin framework https://github.com/http4k/http4k/blob/c619c5ff3194c947bf2b53760efea08a3b93a8d7/http4k-core/src/main/kotlin/org/http4k/core/cookie/Cookie.kt#L12

Reply:
Thanks for sharing these references, really helps me understand